marray/__init__.py

```diff
@@ -459,12 +449,20 @@ def get_sort_fun(name):
         def sort_fun(x, /, *, axis=-1, descending=False, stable=True):
             x = asarray(x)
             data = xp.asarray(x.data, copy=True)
-            sentinel = info(x).min if descending else info(x).max
+            sentinel = _xinfo(x).min if descending else _xinfo(x).max
+            any_masked = xp.any(x.mask)
+            if any_masked and xp.any((data == sentinel) & ~x.mask):
+                minmax = "minimum" if descending else "maximum"
+                message = (f"The {minmax} value of the data's dtype is included in the "
+                           "non-masked data; this complicates sorting when values "
+                           "are present. Consider promoting to another dtype to use "
```

Comment:
Could you explain what you mean by "when values are present"? Any values? Specifically the minmax values? Masked values?

Reply:
Oops:
```suggestion
                message = (f"The {minmax} value of the data's dtype is included in the "
                           "non-masked data; this complicates sorting when masked values "
                           "are present. Consider promoting to another dtype to use "
```